src/main/java/uk/gov/companieshouse/api/accounts/validation/AccountsValidator.java

```diff
@@ -25,19 +29,33 @@
 
     private final SmallFullService smallFullService;
 
-    private final CurrentPeriodTxnClosureValidator currentPeriodTnClosureValidator;
+    private final CurrentPeriodTxnClosureValidator currentPeriodTxnClosureValidator;
 
-    private final PreviousPeriodTxnClosureValidator previousPeriodTnClosureValidator;
+    private final PreviousPeriodTxnClosureValidator previousPeriodTxnClosureValidator;
+
+    private final StocksTxnClosureValidator stocksTxnClosureValidator;
+
+    private final CurrentPeriodService currentPeriodService;
+
+    private final PreviousPeriodService previousPeriodService;
 
     @Autowired
-    public AccountsValidator(CompanyService companyService, CompanyAccountService companyAccountService,
-            SmallFullService smallFullService, CurrentPeriodTxnClosureValidator currentPeriodTnClosureValidator,
-            PreviousPeriodTxnClosureValidator previousPeriodTnClosureValidator) {
+    public AccountsValidator(CompanyService companyService,
```

Comment:
This is being flagged as a code smell by Sonar. Did you tell Sonar to ignore it?

Reply:
This is now fixed on Sonar